kotlinx.fuzz.engine/src/main/kotlin/kotlinx/fuzz/KFuzzEngine.kt

```diff
@@ -1,12 +1,15 @@
 package kotlinx.fuzz
 
 import java.lang.reflect.Method
+import kotlinx.fuzz.reproduction.CrashReproducerWriter
+
+abstract class KFuzzEngine {
```

Comment:
As I mentioned in my reply, we don't need to convert this interface to an abstract class. We can just add a property to it

Reply:
Fixed